kitsune/questions/cron.py

```diff
@@ -186,9 +186,11 @@ def report_employee_answers():
     num_answered = questions.filter(num_answers__gt=0).count()
 
     # Total number of questions answered by user in tracked_group
-    num_answered_by_tracked = Answer.objects.filter(
-        question__in=questions,
-        creator__in=tracked_users).values_list('question_id').distinct().count()
+    num_answered_by_tracked = {}
+    for user in tracked_users:
+        num_answered_by_tracked[user.username] = Answer.objects.filter(
+            question__in=questions,
+            creator=user).values_list('question_id').distinct().count()
```

Comment:
So... I tried but it's tricky. We want the number of questions that have an answer by a tracked user (not the number of answers itself, since a question could have multiple answers by the user). I tried this:

```
    num_answered_by_tracked = (Question.objects
        .filter(id__in=questions, answers__creator__in=tracked_users)
        .values('creator__username')
        .annotate(Count('creator')))
```

But that returns and groups the question creator, not the tracked user. And I got stuck there.